README.md

```diff
@@ -1,9 +1,25 @@
-# ARCHILATOR
+# 🅰🆁🅲🅷🅸🅻🅰🆃🅾🆁
 
-A simple terminal-based calculator
+A simple terminal-based calculator 🧮
 
-### Dependencies
+## Dependencies
+  - [node.js](nodejs.org)
```

Comment:
Could you specify which version of node you're running?

Reply:
added